modin/pandas/test/utils.py

```diff
@@ -295,6 +295,9 @@
     "str": str,
     "sum mean": ["sum", "mean"],
     "sum df sum": ["sum", lambda df: df.sum()],
+    "axis-wise sum": lambda axis: (
+        axis.iloc[0] + axis.iloc[1] if isinstance(axis, pandas.Series) else axis + axis
```

Comment:
> do we ever test a Series case?

Yes, we do. This test case is a part of [`agg_func` suite](https://github.com/dchigarev/modin/blob/b87302c20f88bd6b52d4cba4a0a761ff40fa0bc9/modin/pandas/test/utils.py#L292-L303) that is [used in Series tests](https://github.com/modin-project/modin/blob/0bdc482d6f1682e103b4c4d7ee7c4d505d2d3b1c/modin/pandas/test/test_series.py#L624-L630) as well.

> I'm able to reproduce the bug in #3746 if I just use axis.iloc[0] instead of the sum.

I'd like to use more than one element here to ensure that we don't cheat [by passing a single row/column](https://github.com/modin-project/modin/issues/3219#issuecomment-958926232) to the func.

> The function name is misleading

Agreed, renamed to the "sum of certain elements"

> I think we should add a comment explaining why this test case is there

Added the following in-code comment:
"The case verify that returning a scalar that is based on a frame's data doesn't cause a problem"

Reply:
Thanks, that all makes sense.